Lib/test/test_asyncio/test_locks.py

```diff
@@ -200,6 +200,42 @@ def test_cancel_race(self):
         self.assertTrue(tb.cancelled())
         self.assertTrue(tc.done())
 
+    def test_cancel_release_race(self):
+        # Issue 32734
+        # Acquire 4 locks, cancel second, release first
+        # and 2 locks are taken at once.
+        lock = asyncio.Lock(loop=self.loop)
+        lock_count = 0
+
+        async def lockit():
+            nonlocal lock_count
+            await lock.acquire()
+            lock_count += 1
+  
+        async def lockandtrigger():
+            await lock.acquire()
+            self.loop.call_soon(trigger)
+          
+        def trigger():
+            t1.cancel()
+            lock.release()
+
+        asyncio.Task(lockandtrigger(), loop=self.loop)
```

Comment:
Please use  `loop.create_task()` for a task creation

Reply:
Done. Keep in mind asyncio.Task was used in the rest of the tests so I used it in order to be consistent.